cmd/edit/cluster/cmd.go

```diff
@@ -48,10 +48,42 @@ var args struct {
 	noProxySlice              []string
 	additionalTrustBundleFile string
 
+	// Autoscaler options
```

Comment:
editing the autoscaler cannot be done from the cluster object, so it should only be possible from the dedicated autoscaler resource (meaning only by ``dlt/autoscaler/``